src/components/widgets-settings/params-picker/params-picker.directive.coffee

```diff
@@ -8,20 +8,29 @@ module.controller('SettingParamsPickerCtrl', ($scope) ->
   setting = {}
   setting.key = "params-picker"
   setting.isInitialized = false
+  setting.paramName = $scope.param
```

Comment:
`setting.paramName` is not necessary there: a call can be done to `setting.toMetadata()` to identify the param name. For example, 
  ```
    setting.toMetadata()
    -----
    =>
      {
        status_selection: {
          values: ['Status1', 'Status2'],
          reach: 'dashboard'
        }
      }
  ```
  There, `_.keys(setting.toMetadata())[0]` will return `status_selection` which is our param name